python/py-milc/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-milc
+version             1.6.6
+revision            0
+
+license             MIT
+supported_archs     noarch
+platforms           {darwin any}
+
+maintainers         {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description         Opinionated Batteries-Included Python 3 CLI Framework.
+long_description    MILC is a framework for writing CLI tools. It's goal is \
+                    to make getting started easy and to grow with your program \
+                    as it grows. MILC is Batteries Included- it gives you all \
+                    the functionality that your users demand out of the box. \
+                    Argument parsing, configuration files, flexible and \
+                    configurable log output, ANSI colors, spinners, and other \
+                    nicities are combined into one easy to use module.
+
+homepage            https://github.com/clueboard/milc
+
+checksums           rmd160  a574fe3c9e64b1b2172c570fe740774c87ca24db \
+                    sha256  a4a1673718aaceefeb62c1799e48825bc6f4e56bfd8ad4a8e341a7622e6ff000 \
+                    size    24873
+
+python.versions     310 311
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+        port:py${python.version}-setuptools
+
+    depends_run-append \
+        port:py${python.version}-appdirs \
+        port:py${python.version}-argcomplete \
+        port:py${python.version}-colorama \
+        port:py${python.version}-halo
```

Comment:
It might be worth explicitly adding that milc depends on spinners (see [here](https://github.com/clueboard/milc/blob/6896f13ea66da90c0b5799f81bc7955b9642d608/milc/milc.py#L25)).

```python
from spinners.spinners import Spinners
```

I understand that milc depends on halo, which itself depends on spinners, but that could always change in the future.

Reply:
py-milc depends on py-halo, which already has explicitly dependencie on py-spinners

Reply:
does it counts or not?

Reply:
My thinking is that what if py-halo one day no longer requires py-spinners? It's better to be explicit rather than rely on an implicit chain of dependencies.

Reply:
> if py-halo one day no longer requires py-spinners?

I doubt it

Reply:
I think, it is counterproductive fixing non existing issues

Reply:
both py-halo and py-spinners are developed by the same person, py-halo unusable withot py-spinners.